libc/src/stdio/printf_core/CMakeLists.txt

```diff
@@ -8,28 +8,8 @@ add_header_library(
     libc.src.__support.FPUtil.fp_bits
 )
 
-add_object_library(
+add_header_library(
```

Comment:
the cmake changes look good, but since printf is also in the bazel you'll also need to modify that.

The change you need to make is on line 2691 of `llvm-project/utils/bazel/llvm-project-overlay/libc/BUILD.bazel`, you just need to remove the `srcs` member of the `printf_parser` target. Do the same to the `printf_mock_parser` target for now, it's technically unnecessary to keep around but if it's removed right now it'll break a target downstream.